packages/hyperion-autologging/src/ALUIEventPublisher.ts

```diff
@@ -155,7 +161,7 @@ export function publish(options: InitOptions): void {
       }
       const eventData: ALUIEventCaptureData = {
         domEvent: event,
-        event: eventName,
+        event: (eventName as any),
```

Comment:
would the following work?
`eventName as ALUIEvent['event']`

Slightly better than any.

Reply:
Doesn't seem to work,  then it says `domEvent` is incompatible
<img width="712" alt="image" src="https://user-images.githubusercontent.com/6064408/228564979-06ec32af-08ea-45a5-9ab8-a133c2c3da0d.png">